inc/saiswitch.h

```diff
@@ -580,6 +580,21 @@ typedef enum _sai_switch_tunnel_attr_t
 
 } sai_switch_tunnel_attr_t;
 
+/**
+ * @brief Attribute data for
+ *        SAI_SWITCH_ATTR_SUPPORTED_HOSTIF_OPER_STATUS_UPDATE_MODE and
+ *        SAI_SWITCH_ATTR_HOSTIF_OPER_STATUS_UPDATE_MODE.
+ */
+typedef enum _sai_hostif_oper_status_update_mode_t
+{
+    /** Application updates the operational status of hostif */
```

Comment:
The enums name should begin with SAI\_SWITCH\_HOSTIF\_OPER\_STATUS\_UPDATE\_MODE\_XXXX.

Examples of previous such attributes:

sai1.3/saiswitch.h:typedef enum \_sai\_switch\_switching\_mode\_t  
sai1.3/saiswitch.h:    /\*\* Cut-through switching mode \*/  
sai1.3/saiswitch.h:    SAI\_SWITCH\_SWITCHING\_MODE\_CUT\_THROUGH,  
sai1.3/saiswitch.h:    /\*\* Store-and-forward switching mode \*/  
sai1.3/saiswitch.h:    SAI\_SWITCH\_SWITCHING\_MODE\_STORE\_AND\_FORWARD  
sai1.3/saiswitch.h:} sai\_switch\_switching\_mode\_t;  
sai1.3/saiswitch.h:     \* @brief Switching mode  
sai1.3/saiswitch.h:     \* @type sai\_switch\_switching\_mode\_t  
sai1.3/saiswitch.h:     \* @default SAI\_SWITCH\_SWITCHING\_MODE\_STORE\_AND\_FORWARD  
sai1.3/saiswitch.h:    SAI\_SWITCH\_ATTR\_SWITCHING\_MODE,